ledger-core/v2/go.mod

```diff
@@ -19,7 +19,6 @@ require (
 	github.com/golang/protobuf v1.3.2
 	github.com/google/gofuzz v1.0.0
 	github.com/google/gops v0.3.6
-	github.com/google/uuid v1.1.1
```

Comment:
still very strange, we use uuid in various ways

Reply:
it's result of tidy

Reply:
@sexyhamster fixed with running go.mod tidy on fresh master